test/cbmc/proofs/run-cbmc-proofs.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python3
```

Comment:
Is this just the generic run-cbmc-proofs script?  Is there anything special about this version?  Can you comment and hightlight it so even I can find it?

Reply:
Good idea, thank you! I have commented on the changes, starting with the string `# PROJECT SPECIFIC`, and will start to do this for all the other projects also. That should be very helpful if we ever decide to make a unified script that gets added to the starter kit!

In summary, this script differs because it is setting the Kissat environment variable when invoking Make. So there's some checking around that. The user can use the native solver (minisat) if they like, by passing an explicit command-line flag.